src/components/DynamicFormContainer/DynamicForm/CustomTemplates/CustomTextareaWidget.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent, FunctionComponent } from "react";
+import { WidgetProps } from "react-jsonschema-form";
+
+export const CustomTextareaWidget: FunctionComponent<WidgetProps> = ({
+  id,
+  options,
+  placeholder,
+  value,
+  required,
+  disabled,
+  readonly,
+  autofocus,
+  onChange,
+  onBlur,
+  onFocus,
+}) => {
+  const _onChange: (event: ChangeEvent<HTMLTextAreaElement>) => void = ({ target: { value } }) => {
+    return onChange(value === "" ? options.emptyValue : value);
+  };
```

Comment:
felt that this particular file's code writing style was out of place, think u may want to put a comment at the very top, linking to the source code which u referred to?

Reply:
ok will add it in